fala/apps/adviser/tests/test_adviser_extras.py

```diff
@@ -1,6 +1,6 @@
 import unittest
 
-from adviser.templatetags import adviser_extras
+from fala.apps.adviser.templatetags import adviser_extras
```

Comment:
I think my comment got lost: this shouldn't be necessary and is an artefact of the IDE not having a proper Django runtime